comp/dogstatsd/server/parse_events.go

```diff
@@ -44,8 +44,8 @@ type dogstatsdEvent struct {
 }
 
 type eventHeader struct {
-	titleLength int
-	textLength  int
+	titleLength int64
```

Comment:
CodeQL flagged this. Not sure why it was not previously flagged. Just want to draw extra attention to this change since this is a big PR.

Reply:
I think we should leave it as it were, and change ParseInt64 to ParseInt instead. Can get rid of `titleLength > int64(math.MaxInt)` while at it too.